locales/en.yml

```
@@ -63,6 +63,16 @@ eventView:
     dashboard: Dashboard
     edit: Edit
     share: Share
+  EventDashboard:
+    noAccess: Only event owner can access this area.
+    unlink: Unlink
+    couple: Couple
+    cancel: Cancel
+    checkIn: CheckIn
+    rsvp: RSVP
+    subs: Subscribe
```

Better to use full words, so that translators can understand the context better. Avoid shortcuts. Same for variables in the code FYI ;)

```suggestion
    subscribe: Subscribe
```